src/map/status.c

```diff
@@ -12089,15 +12088,22 @@ static int status_change_timer(int tid, int64 tick, int id, intptr_t data)
 			break;
 
 		case SC_CASH_BOSS_ALARM:
-			if( sd && --(sce->val4) >= 0 ) {
+			if (sd != NULL && --(sce->val4) >= 0) {
 				struct mob_data *boss_md = map->id2boss(sce->val1);
-				if( boss_md && sd->bl.m == boss_md->bl.m ) {
-					clif->bossmapinfo(sd->fd, boss_md, 1); // Update X - Y on minimap
-					if (boss_md->bl.prev != NULL) {
-						sc_timer_next(5000 + tick, status->change_timer, bl->id, data);
+				if (boss_md != NULL) {
+					if (sd->bl.m != boss_md->bl.m) // Changed map
```

Comment:
if there is 1 statement in curly bracket, all other statements of those chain conditions also needs curly bracket
```
					if (sd->bl.m != boss_md->bl.m) { // Changed map
 						return 0;
					} else if (boss_md->bl.prev != NULL) { // Boss monster still alive, update x and y position on map
						sce->val2 = 0;
						clif->bossmapinfo(sd->fd, boss_md, BOSS_INFO_ALIVE);
					} else if (boss_md->spawn_timer != INVALID_TIMER && sce->val2 == 0) { // Boss monster is dead
						sce->val2 = 1;
						clif->bossmapinfo(sd->fd, boss_md, BOSS_INFO_DEAD);
						return 0;
					}
```
https://github.com/HerculesWS/Hercules/wiki/Coding-Style#chapter-3-placing-braces-and-spaces
> This does not apply if only one branch of a conditional statement is a single statement; in the latter case use braces in both branches: